client/app/visualizations/cohort/index.js

```diff
@@ -115,17 +115,21 @@ function prepareSimpleData(sortedData, options) {
 function prepareData(rawData, options) {
   rawData = _.map(rawData, item => ({
     date: item[options.dateColumn],
-    stage: item[options.stageColumn],
-    total: item[options.totalColumn],
-    value: item[options.valueColumn],
+    stage: parseFloat(item[options.stageColumn]),
```

Comment:
`stage` probably can be even `integer`

Reply:
https://github.com/getredash/redash/pull/3397/commits/655ce506c82b3165985c1e45b8f4fc3ce45b4c10